<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line</class>
 <widget class="QWidget" name="Line">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Flags</string>
   </property>
   <widget class="QCheckBox" name="flag0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Line power</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flag1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>High voltage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flag2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Falling edge</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flag3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge triggered</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Test signal</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Correlator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Correlator</string>
   </property>
   <widget class="QPushButton" name="TakeDark">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Dark</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y scale:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="YScale">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Linear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Log</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip lines:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="StartLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="Progress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>114</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
